### ISSUE TITLE
3' overhang is missing a glpyh

### ISSUE THREAD

**Issue by cjmyers** [VisBOL/visbol-js#65]

**Comment by cjmyers** [comment]
Seems correct but Dany reports it was flipped.

**Comment by dany-fu** [comment]
@cjmyers 
This does appear correct using the test file you sent (wholeShootingMatch). The issue is with SBOLDesigner (v. pre-release 3.0) - the files generated there are flipping the SO terms. 

````
  <sbol:ComponentDefinition rdf:about="http://www.dummy.org/_3OH/1">
    <sbol:persistentIdentity rdf:resource="http://www.dummy.org/_3OH"/>
    <sbol:displayId>_3OH</sbol:displayId>
    <sbol:version>1</sbol:version>
    <prov:wasGeneratedBy rdf:resource="http://www.dummy.org/UnnamedPart_SBOLDesignerActivity/1"/>
    <sbol:type rdf:resource="http://www.biopax.org/release/biopax-level3.owl#DnaRegion"/>
    <sbol:role rdf:resource="http://identifiers.org/so/SO:0001932"/>
  </sbol:ComponentDefinition>
  <sbol:ComponentDefinition rdf:about="http://www.dummy.org/_5OH/1">
    <sbol:persistentIdentity rdf:resource="http://www.dummy.org/_5OH"/>
    <sbol:displayId>_5OH</sbol:displayId>
    <sbol:version>1</sbol:version>
    <prov:wasGeneratedBy rdf:resource="http://www.dummy.org/UnnamedPart_SBOLDesignerActivity/1"/>
    <sbol:type rdf:resource="http://www.biopax.org/release/biopax-level3.owl#DnaRegion"/>
    <sbol:role rdf:resource="http://identifiers.org/so/SO:0001933"/>
  </sbol:ComponentDefinition>
````

3' SHOULD be 1933 and 5' SHOULD be 1932

**Comment by cjmyers** [comment]
@dany-fu please grab a new version of SBOLDesigner.  I believe these are fixed in the final 3.0 release.
